recipes/tardis/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "tardis" %}
+{% set version = "0.5.8" %}
+{% set sha256 = "8918af5f4e76ebd78b10aba48ceb5eadb8d544153d4bb746ee8a8c0b23f711bd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/AgResearch/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+  entry_points:
+    - tardis = tardis.run:tardis_main
+
+requirements:
+  build:
+    - python 2.7.*
+    - setuptools
+    - setuptools_scm
+    - setuptools_scm_git_archive
+    - make
+    - gcc  # [not osx]
+    - llvm # [osx]
+    - zlib {{CONDA_ZLIB}}*
+
+  run:
+    - python 2.7.*
+    - libgcc # [not osx]
+    - zlib {{CONDA_ZLIB}}*
+
+test:
+  imports:
+    - tardis
+  source_files:
+    - kseq_split/test.fastq
+  commands:
+    - tardis -h
+    - kseq_split -s 0.1 -o fastq test.fastq 10 split-%05d.fastq
```

Comment:
I'm surprised this works . . . @bgruening does mulled-build copy over test files from source now?

Reply:
Ah, I see kseq_split is returning a success exit code even when it can't find its input files.  Fixed now.